<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget_frames">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_add_me_noscroll">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="frame_add_me_scroll">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>191</red>
                <green>191</green>
                <blue>191</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_viewMessages"/>
    <addaction name="action_viewSequencer"/>
    <addaction name="action_viewRealtime_props"/>
    <addaction name="action_viewRealtime_plots"/>
    <addaction name="action_connections"/>
    <addaction name="action_viewParameters"/>
   </widget>
   <widget class="QMenu" name="menuRuntime_Parameters">
    <property name="title">
     <string>Runtime Parameters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuProperty_Pool">
     <property name="title">
      <string>System Property Pool</string>
     </property>
    </widget>
    <addaction name="menuProperty_Pool"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuView"/>
   <addaction name="menuRuntime_Parameters"/>
   <addaction name="menuSettings"/>
   <addaction name="menuApplication"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_viewMessages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+M</string>
   </property>
  </action>
  <action name="action_viewSequencer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sequencer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_viewRealtime_props">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RT Viewer Properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="action_viewParameters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="action_viewRealtime_plots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RT Viewer Plots</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+P</string>
   </property>
  </action>
  <action name="action_connections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="jvx-qt-host.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_viewMessages</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewMessages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_viewParameters</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_viewRealtime_props</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewRealtime_props()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_viewSequencer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewSequencer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_viewRealtime_plots</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewRealtime_plots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveConfigFileAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_connections</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>triggered_viewConnections()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveConfigFile()</slot>
  <slot>triggered_viewMessages()</slot>
  <slot>triggered_viewParameters()</slot>
  <slot>triggered_viewSequencer()</slot>
  <slot>triggered_viewRealtime_props()</slot>
  <slot>openSequenceEditor()</slot>
  <slot>triggered_viewRealtime_plots()</slot>
  <slot>triggered_skipStateSelected()</slot>
  <slot>triggered_toggleCryptConfig()</slot>
  <slot>saveConfigFileAs()</slot>
  <slot>openConfigFile()</slot>
  <slot>triggered_viewConnections()</slot>
 </slots>
</ui>
